modules/friendlytag.js

```diff
@@ -267,7 +267,7 @@ Twinkle.tag.callback = function friendlytagCallback() {
 			numAdded: 0,
 			numRemoved: 0
 		};
-		$(Window.buttons[0]).after(statusNode);
+		$('.submitButtonProxy').after(statusNode);
```

Comment:
The purpose of using `Window.buttons[0]` was to specifically target the submit button of TW Tag window, even if there are other Morebits simpleWindows open. 

Since #852 allows for adding a class name to the quickform submit, that should be done here for uniqueness (maybe `tw-tag-submit`), rather than going with submitButtonProxy.

Reply:
Huh!  I hadn't even considered the possibility of someone doing that.  Updated with that in mind.